packages/kbn-unified-data-table/src/components/custom_control_columns/additional_row_control/row_control_column.tsx

```diff
@@ -37,28 +37,39 @@ export const RowControlCell = ({
         label,
         onClick,
         tooltipContent,
+        ...extraProps
       }) => {
-        return (
-          <EuiToolTip
-            content={tooltipContent ?? label}
-            delay="long"
-            anchorClassName="unifiedDataTable__rowControl"
-          >
-            <EuiButtonIcon
-              data-test-subj={dataTestSubj ?? `unifiedDataTable_rowControl_${props.columnId}`}
-              disabled={disabled}
-              iconSize="s"
-              iconType={iconType}
-              color={color ?? 'text'}
-              aria-label={label}
-              onClick={() => {
-                if (record) {
-                  onClick?.({ record, rowIndex });
-                }
-              }}
-            />
-          </EuiToolTip>
+        const control = (
+          <EuiButtonIcon
+            aria-label={label}
+            className="unifiedDataTable__rowControl"
+            color={color ?? 'text'}
+            data-test-subj={dataTestSubj ?? `unifiedDataTable_rowControl_${props.columnId}`}
+            disabled={disabled}
+            iconSize="s"
+            iconType={iconType}
+            onClick={() => {
+              if (record && onClick) {
+                onClick({ record, rowIndex });
+              }
+            }}
+            {...extraProps}
+          />
         );
+
+        if (tooltipContent) {
```

Comment:
Consider adding a unit test for this. Thanks.

Reply:
Make sense, added with [refactor(discover): add test for tooltip](https://github.com/elastic/kibana/pull/195630/commits/cac0f2ab2ed3c1a60abef6150d94caf1c6ba3368)